data/en/isdateobject.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name":"isDateObject",
+	"type":"function",
+	"syntax":"isDateObject(value)",
+	"returns":"boolean",
+	"related":["isDate","isObject"],
+	"description":"Checks if the given value is a date / time object. The difference between isDateObject and isDate, latter returns true for date / time objects as well as date / time strings.",
+	"params": [
+		{"name":"value","description":"A value, usually the name of a variable","required":true,"default":"","type":"variableName","values":[]}
```

Comment:
Type is `any`. The description is erroneously referring to the name of a variable, which I've just brought up to Adobe since the description is copied from their documentation. I've also brought up the fact that the param name should be `dateobject`.

Reply:
Have made the change and committed.